parsec/backend/cli/run.py

```diff
@@ -258,6 +258,21 @@ def handle_parse_result(self, ctx, opts, args):
     envvar="PARSEC_ADMINISTRATION_TOKEN",
     help="Secret token to access the administration api",
 )
+@click.option(
+    "--invite-mail-server", envvar="PARSEC_INVITE_SERVER", help="SMTP server for the invite mail"
```

Comment:
envvar and command parameter should have the same naming pattern (given env var name is not printed when doing `--help`, that's the convention we came with)

Beside, I think we should name this `--mail-...`/`PARSEC_MAIL_...` instead of `--invite-mail-...`/`PARSEC_INVITE_MAIL_...`, given this configuration is about sending mails (and maybe we will have other uses for sending mails apart from invitation)